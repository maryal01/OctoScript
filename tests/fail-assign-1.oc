int x = 12;
x = "twelve";