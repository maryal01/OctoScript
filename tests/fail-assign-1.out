Fatal error: exception Failure("Illegal assignment of int and string")